src/functions/symbolsOrd.js

```diff
@@ -37,7 +39,7 @@ defineFunctionBuilders({
     htmlBuilder(group, options) {
         return buildCommon.makeOrd(group, options, "textord");
     },
-    mathmlBuilder(group, options) {
+    mathmlBuilder(group: ParseNode<"textord">, options) {
```

Comment:
Why these generic types can't be inferred from `type`?

Reply:
IMO, it should be able to, but it's confused by `getVariant(group, options)`. Since the latter takes `group`s of a variety of types, here it might be inferring that even though `type === "textord"`, `type: SymbolNodeType` where `SymbolNodeType` might be defined to be all possible types encompassed by `SymbolParseNode`.